src/main/java/io/passport/server/model/Organization.java

```diff
@@ -0,0 +1,25 @@
+package io.passport.server.model;
+
+import lombok.Getter;
+import lombok.Setter;
+import javax.persistence.*;
+
+/**
+ * Organization model for later use, implemented early to implement Study structure properly.
+ */
+@Entity
+@Table(name = "organizations")
+@Getter
+@Setter
+public class Organization {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "organization_id")
+    private String organizationId;
```

Comment:
I could not understand the purpose of this field. We already have id field which is uniquely identifies the Organization entity right ? I think we can remove organizationId field which is redundant.